tests/core/test_api/test_endpoints.py

```diff
@@ -0,0 +1,8 @@
+from ai_transform.api.client import Client
+
+
+class TestEndpoints:
+    def test_generic_request(self, test_client: Client):
+        test_client.api.post("/auth/users/list")
```

Comment:
This doens't actually test the response object. can we chekc the type that's returned here?

Reply:
ALso if this errors - we have no way of knowing atm.